devfiles/python/devfile/devfile.yaml

```diff
@@ -31,7 +31,7 @@ components:
                 emptyDir: {}
             containers:
               - name: jupyter
-                image: 'mas.dit.maap-project.org/root/maap-workspaces/jupyterlab/vanilla:develop'
+                image: 'mas.dit.maap-project.org/root/maap-workspaces/jupyterlab/python :develop'
```

Comment:
Can we remove this extra space after `python`?

Reply:
@grallewellyn Fixed!